unit-tests/func/serializable-device/test-json-load.py

```diff
@@ -0,0 +1,51 @@
+import sys
+sys.path.insert(0,"C://Users//mmirbach//git//librealsense//build//RelWithDebInfo")
+sys.path.insert(0,"C://Users//mmirbach//git//librealsense//unit-tests//py")
+
+from time import sleep
+import pyrealsense2 as rs, test
+
+devices = test.find_devices_by_product_line_or_exit(rs.product_line.L500)
```

Comment:
If it's only L500, should we signify that in the filename, somehow?
Is there also a test for the D400?

Reply:
will change to test-l500-json-load
I didn't write one for d400, if you tell me how it is supposed to work I can make one

Reply:
You don't have a D400, do you?

Reply:
no I do not